### DISCUSSION TITLE
Добавить машину для полиции

### DISCUSSION THREAD

**Discussion by octothorpteam** [octothorp-team/public-dobrograd#559] [Предложения]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/billblack/32/763_2.png) [BillBlack](https://forum.octothorp.team/u/BillBlack) ([#4944](https://forum.octothorp.team/p/4944))

**Мое предложение немного интересное по своему смыслу, но совсем не сложное.**
 Насколько я знаю, в GTAIV у полиции было всего 2 машины - Merit и Buffalo, а Vapid является крузером NOOSE. Также у NOOSE есть Patriot. Как мы знаем, патриот - это довольно устойчивая и крепкая машина, которую зачастую не так просто остановить.
 В карпаке с машинами GTA:SA у полиции было, если не ошибаюсь, четыре машины. Вот и сейчас я предлагаю впихнуть четвертую.
 А именно Patriot в полицейской комплектации. Он даст дополнительное преимущество офицерам полиции при перекрытии дорог, а также поможет, как таран для остановки при пит-маневрах.
 Выполнить это нетрудно. Раз уж вы NOOSE Cruiser переделали один, со вторым проблем не должно особых быть.
 А выдачу ее можно сделать только для Добродеев, чтобы никто не злоупотреблял мощью. Или просто безумно дорогую аренду.

Как вы считаете, нужна такая машина полиции?

- Да, нужна;
- Нет, не нужна.

0голосов

Также я прошу оставить конструктивную критику и обоснование.
 Сделал все, как сказали, без лишнего пафоса и агитации в вариантах ответа.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/arn/32/272_2.png) [Arn](https://forum.octothorp.team/u/Arn) ([#4962](https://forum.octothorp.team/p/4962))


Тогда все будут без причины просто брать огромный внедорожник и таранить машинки за полицию ![:slight_smile:](https://forum.octothorp.team/images/emoji/twitter/slight_smile.png?v=10 ":slight_smile:") Против. Иногда тоже самое случалось и с Enforcer в старом карпаке, так что это будет и с патриотом.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/billblack/32/763_2.png) [BillBlack](https://forum.octothorp.team/u/BillBlack) ([#4980](https://forum.octothorp.team/p/4980))


Поэтому я предлагаю делать неподъемную цену аренды, чтобы неповадно было разбивать зазря такую машину.
 Кроме того, если брать в учет буффало, который все берут и гоняют на нем под 100 миль, его тоже можно убрать, а то гоняют и врезаются.
 Я понимаю, что это странно, но все же, я не вижу, чтобы полиция просто так таранила машины. Те, кто таранят, они смогут это делать и на уже имеющихся машинах, а полиции все же необходимо какое-то преимущество. По-моему карпак полиции не имеет никаких фишек относительно карпака гражданских.
 Я ни разу не видел, чтобы кто-то таранил машины без причины на крузерах. И кстати хочу отметить для справедливости, а то ты вот говоришь “иногда случалось”. Так вот, справедливости ради, скажу, что Enforcer никто никогда и не брал почти.

Мне кажется, что иногда гражданские сильно преувеличивают вред полиции. Зачастую они сами неправы, но вешают собак на полицию.
 Те, кто таранят машины - их можно и банить. Вот так вот (капитан Очевидность).

Так что вряд ли это сильно повлияет прямо на РП. Чтобы никто ничего не нарушал, надо сервер закрывать. Иначе никак.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/обжора/32/247_2.png) [Обжора](https://forum.octothorp.team/u/Обжора) ([#5025](https://forum.octothorp.team/p/5025))


Патриот изначально был в карпаке ,но его убрали по причине того что он никак не вписывается в сеттинг Доброграда, а вапид к слову есть в двух вариантах Noose и LCPD ,на сервере вариант NOOSE

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/обжора/32/247_2.png) [Обжора](https://forum.octothorp.team/u/Обжора) ([#5027](https://forum.octothorp.team/p/5027))


На кой удалять Буффало если он только для Добродеев , и в предачу он хотябы вписывается в сеттинг.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/kustik/32/467_2.png) [Kustik](https://forum.octothorp.team/u/Kustik) ([#5047](https://forum.octothorp.team/p/5047))


Против.

Даже если эту машину сделать дорогой, в плане аренды, то она:
 а) Плохо впишется в сеттинг Доброграда;
 б) Её по-возможности будут брать все, а значит кол-во ненужных таранов увеличиться;
 в) 3 Крузеров полиции и так хватает;

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/4cb/32/176_2.png) [4CB](https://forum.octothorp.team/u/4CB) ([#5052](https://forum.octothorp.team/p/5052))


Патриот это что-то на уровне SWAT или военных. Раньше был Rancher(двухместный с высокой подвеской внедорожник) в паке GTA:SA, можно найти альтернативу Ранчеру и добавить его, но никак не военный хаммер.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/mr.nekita/32/296_2.png) [mr.nekita](https://forum.octothorp.team/u/mr.nekita) ([#5245](https://forum.octothorp.team/p/5245))


**Против.**
 Он был, но его убрали. Не вписывается, да и он слишком имбовый.


И разработчики щас делают системы персонажей, организаций, аксессуаров, так что не думаю что в планах что-то из предложений еще месяца два.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/gasw1n/32/2386_2.png) [gasw1n](https://forum.octothorp.team/u/gasw1n) ([#5252](https://forum.octothorp.team/p/5252))


Его убрали не из за дисбаланса. А из за того что это патриот, ясное дело он не будет в автопарке полиции. Ясное дело его не вернут. Вапид мерит буффало что ещё добавить? Броневик сват? Самолет? Вертолёт?

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/respect_з/32/1736_2.png) [Respect_З](https://forum.octothorp.team/u/Respect_З) ([#6118](https://forum.octothorp.team/p/6118))


Можно сделать эту машину только для кэпа

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/billblack/32/763_2.png) [BillBlack](https://forum.octothorp.team/u/BillBlack) ([#6119](https://forum.octothorp.team/p/6119))


Кстати-ка. Неплохая мысль. Правда и кэпы иногда бывают теми еще фруктами, но допустим. Можно и так. Потому что остановить патриот вапидом нереально почти. А вот Patriot VS Patriot - уже что-то.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/respect_з/32/1736_2.png) [Respect_З](https://forum.octothorp.team/u/Respect_З) ([#6123](https://forum.octothorp.team/p/6123))


И сделать машину для добродея ![:ok_hand:](https://forum.octothorp.team/images/emoji/twitter/ok_hand.png?v=10 ":ok_hand:")

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/killerbunny/32/542_2.png) [KillerBunny](https://forum.octothorp.team/u/KillerBunny) ([#6500](https://forum.octothorp.team/p/6500))


**ЗА**

* * *

В кар паке Доброграда есть большие машины, такие, как тот же Патриот. И как вы его собираетесь останавливать? Чтобы остановить большую машину, нужна большая машина. Сейчас, чтобы чувствовать себя в полной безопасности и безнаказанности на дороге достаточно просто купить Патриот. Чтобы остановить такую машину собирается чуть ли не весь ПД. И на остальные вызовы людей просто не хватает. И то не факт, что остановят. Поэтому я предлагаю:

- Добавить полицейский внедорожник (или что-то типо того, как на картинке)
![image](https://forum.octothorp.team/uploads/default/original/2X/1/12481b9a38ec5f928159c20782dbc6becad36a69.jpeg)
- Разрешить арендовывать его только Лейтенантам полиции и квалифицированным сотрудникам DPD.
- Сделать цену аренды ~65.000-85.000руб, чтобы на этих машинах просто так не ездили


Многие эти идеи были и выше ![:point_up:](https://forum.octothorp.team/images/emoji/twitter/point_up.png?v=10 ":point_up:"), но я их объединил в одно и вставил свои 5 копеек мнения.

**Reply by LENFYS** [reply]
Нет, парень, во всех департаментах Штатов автомобили выдаються вне зависимости от звания, а по мере их использованости и устарелости.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/jarvis/32/397_2.png) [JaRviS](https://forum.octothorp.team/u/JaRviS) ([#6510](https://forum.octothorp.team/p/6510))


Про цену аренды - помимо муниципальной полиции есть также и дпд, которые также могут просто так начать тебя таранить, при этом у дпд нету залога за автомобиль, ну как бы есть но он равен нулю. Так что это спасёт и то, только наполовину
 UPD: Ну хантли, я думаю это не канонично будет для американской полиции, зашел бы какой-нибудь landstalker, ака форд экспедишн, ну а так, честно ни разу не видел чтобы крузеры в лесу застревали, я даже специально первую машину на новом карпаке купил ранчер, чтобы от полиции уезжать, но они даже по холмам на скорости хуячат и все.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/billblack/32/763_2.png) [BillBlack](https://forum.octothorp.team/u/BillBlack) ([#6570](https://forum.octothorp.team/p/6570))


UPD, ибо редактировать уже не могу.
**Ну или действительно хотя бы внедорожник, потому что сейчас ездить на крузерах по лесу или тем более по горам с обратной стороны трейлер-парка почти не представляется возможным. Да, патриот может и лишний, но что-нибудь вроде того же Huntley, потому что для погони по лесу, что часто решают сделать люди на таких машинах, нужна машина с высокой подвеской и, кстати, улучшенными характеристиками. То есть Huntley, например, с более высокой скоростью поворота и более мощным двигателем. Сейчас люди уже не берут скоростью, а хитростью и даже на Futo суются в лес, а наши крузеры до сих пор застревают в корнях деревьев.**
